### ISSUE TITLE
Need Help Setting Keys in Zbrush 

### ISSUE THREAD

**Issue by oggy3d** [dumbeau/AutoHotPie#162]
in zbrush i want to add brushes like move brush its shortcut key is B+M+V but in Autohotkey menu it cant let me add more than 1 key like its only letting me press B need help plzz been looking for help

**Comment by Da3n0n** [comment]
Yeah you are right this is the way it currently works there is no sequence of keys it’s modifiers and keys that’s all , 

two potential workarounds though 

- change the shortcut in zbrush 
- download macro tool set macro for  F23 to be  B + M + V  and then set in AutoHotPie the slice to be F23 when F23 is pressed it will fire the macro you made for B M V

**Comment by Forikundo** [comment]
You can also create a script in autohotkey and set autohotpie to run the script. I do that with tons of stuff

**Comment by oggy3d** [comment]
> You can also create a script in autohotkey and set autohotpie to run the script. I do that with tons of stuff

you got discord server i want to talk to you

**Comment by shalock999** [comment]
You can set it in AutoHotPie like this
Function->Send Key->B->Add Keystroke->M->Add Keystroke->V
then modify the value "Time between keystrokes"` about 25~30ms
